streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -1088,6 +1088,28 @@ public void shouldTriggerRecordingOfRocksDBMetricsIfRecordingLevelIsDebug() {
         }
     }
 
+    @Test
+    public void shouldGetClientSupplierFromConfigForConstructor() {
+        final StreamsConfig config = new StreamsConfig(props);
+        final StreamsConfig mockConfig = spy(config);
+        when(mockConfig.getKafkaClientSupplier()).thenReturn(supplier);
+
+        new KafkaStreams(getBuilderWithSource().build(), mockConfig);
+        // It's called once in above when mock
+        verify(mockConfig, times(2)).getKafkaClientSupplier();
```

Comment:
Why `times(2)` but not `times(1)` ?

Reply:
Somehow above `when(mockConfig.getKafkaClientSupplier()).thenReturn(supplier);` is counted.